src/main/resources/templates/csdept/classroom/fragments/title.html

```diff
@@ -1,3 +1,3 @@
-<h1>Search For Courses in CS Department Controlled Classrooms</h1>
-<p>This page allows you to search for all courses offered in CS department controlled classrooms in a specific quarter.
+<h1>Search by Classrooms</h1>

```

Comment:
Capitalize "By"?

Reply:
Some folks prefer lowercase.   But the most important thing is consistency.